rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -71,39 +69,29 @@
 
     private Layout root;
 
-    /* displays nodes as a compact grid */
-    private FlowPanel flow;
-
-    boolean _borderSwitch;
+    private boolean _borderSwitch;
 
-    boolean _doNotScroll;
+    private boolean _doNotScroll;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_doNotScroll" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)